snuba/datasets/events.py

```diff
@@ -1,7 +1,20 @@
-from snuba.clickhouse import Array, ColumnSet, DateTime, FixedString, Float, Nested, Nullable, UInt, String
+from snuba.clickhouse import (
+    Array,
+    ColumnSet,
+    DateTime,
+    escape_col,
+    FixedString,
+    Float,
+    Nested,
+    Nullable,
+    String,
+    UInt,
+)
 from snuba.datasets import DataSet
 from snuba.datasets.schema import ReplacingMergeTreeSchema
 from snuba.processor import MessageProcessor, process_message
+from snuba.settings import TIME_GROUP_COLUMNS
+from snuba.util import NESTED_COL_EXPR_RE, tag_expr, tags_expr, time_expr
```

Comment:
Leaving these in utils as I assume tags/contexts/time might be useful across multiple datasets